xml/MTL/2/mtl_aggregator.xml

```diff
@@ -493,63 +493,63 @@
 			<TELC:lsb>15</TELC:lsb>
 			<TELC:msb>17</TELC:msb>
 		</TELC:sample>
-		<TELC:sample name="Atom Core0" datatypeIDREF="tgeneric_sample" sampleID="Atom Core0">
```

Comment:
Last update date has not been updated. I assume that's intentional as the changes are minor?

Reply:
yes, it is minor changes